### DISCUSSION TITLE
Upgrade problem: 4.14.2 module not acknowledged as latest version

### DISCUSSION THREAD

**Discussion by Donaz72** [PrestaShop/PrestaShop#35429] [Q&A] [answered]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Good morning, I wanted to update the prestashop version 1.6.1.13 to the most recent one using the 1-Click Upgrade module but despite having updated the 1-click upgrade module (see image) I am unable to update. How come? What could be the problem?
![PRESTA](https://github.com/PrestaShop/PrestaShop/assets/160619014/b7a15085-4a0b-4993-b73c-e094f7e21f19)

### Expected behavior

update version

### Steps to reproduce

I go to form
exhaust version
module installation

### PrestaShop version(s) where the bug happened

1.6.1.13

### PHP version(s) where the bug happened

7.1

### If your bug is related to a module, specify its name and its version

1-click upgrade  (la tua versione corrente è la v4.14.2)

### Your company or customer's name goes here (if applicable).

Città Immagine snc - SiTimbri

**Comment by Hlavtox** [comment]
@kpodemski @matks problem

**Comment by matks** [comment, marked as answer]
@Donaz72 Latest version of autoupgrade module is 5.0.0 however it does not support 1.6 -> 1.7 migrations because the 1.6 software is too old.

I suggest you download [this patch](https://github.com/PrestaShop/autoupgrade/releases/tag/v4.14.3) which disables, on PrestaShop 1.6 and lower, the system that would push people to download versions 4.16 and higher of the module. This will remove the warning message you can see.

**Reply by Donaz72** [reply]
Thanks, I managed to get all the green ticks but it doesn't update.

**Comment by matks** [comment]
Bug report transformed into Q&A discussion

**Comment by Donaz72** [comment]
Good morning, I did what @matks suggested, the ticks are now all green, continue doing the "VERSION COMPARISON" indicating an error

PrestaShop original version:
Error: Unable to check md5 files

I also checked the folder permissions which are all at 755
![Screenshot 2024-02-21 at 10-38-13 1-Click Upgrade 1-Click Upgrade • Si Timbri E-commerce di Città Immagine snc](https://github.com/PrestaShop/PrestaShop/assets/160619014/e22f9fb4-97ba-4c63-a79f-8fd69faca621)

**Comment by MatShir** [comment]
In french, it should work with the translator https://www.prestashop.com/forums/topic/679589-mise-%C3%A0-jour-error-unable-to-check-md5-files/
Hope it helps 🤗
